src/main/code.ezuino

```diff
@@ -1,6 +1,5 @@
-func int main() {
-    while(1<2){
-        int a
-    }
-    return 1
+int a
+func  b(int a) {
+
 }
+a := b(1)
```

Comment:
Remove this file from commits
Use 
* `git fetch origin` 
* `git checkout origin/master -- src/main/code.ezuino` 
And commit the file